lib/registries/addon/components/license-viewer/styles.scss

```diff
@@ -2,3 +2,11 @@
     padding: 0;
     border: 0;
 }
+
+.LicenseView {
```

Comment:
This is being addressed in a separate PR

Reply:
Futa, I apologize. I had been concurrently working on two tickets. I believe I had switched branches but may not have. Let me make these changes and send them through.